packages/teleport-react-component-generator/src/react-component.ts

```diff
@@ -1,29 +1,26 @@
-import AssemblyLine from '@teleporthq/teleport-assembly-line'
-import Builder from '@teleporthq/teleport-builders'
-import Resolver from '@teleporthq/teleport-resolver'
-import Validator from '@teleporthq/teleport-validator'
+import { AssemblyLine, Builder, Resolver, Validator } from '@teleporthq/teleport-generator-core'
 
 import reactComponentPlugin from '@teleporthq/teleport-react-base-component'
 import reactInlineStylesPlugin from '@teleporthq/teleport-react-inline-styles'
 import reactPropTypesPlugin from '@teleporthq/teleport-react-proptypes'
 import importStatementsPlugin from '@teleporthq/teleport-react-import-statements'
 
-import { createFile } from '@teleporthq/teleport-shared/lib/utils/project-utils'
-import { sanitizeVariableName } from '@teleporthq/teleport-shared/lib/utils/string-utils'
-import { FILE_TYPE } from '@teleporthq/teleport-constants'
+import { createFile } from '@teleporthq/teleport-generator-shared/lib/utils/project-utils'
+import { sanitizeVariableName } from '@teleporthq/teleport-generator-shared/lib/utils/string-utils'
+import { FILE_TYPE } from '@teleporthq/teleport-generator-shared/lib/constants'
 
-import htmlMapping from '@teleporthq/teleport-uidl-definitions/lib/elements-mapping/html-mapping.json'
+import htmlMapping from '@teleporthq/teleport-generator-shared/src/uidl-definitions/elements-mapping/html-mapping.json'
 import reactMapping from './react-mapping.json'
```

Comment:
Like here, if we see line 10 and 12 here, one place we import from lib and another place from src

Reply:
only .json imports are from src because they are not copied by default by typescript, will fix it